setup.py

```diff
@@ -149,4 +149,5 @@ def get_long_description():
               },
           zip_safe=False,
           test_suite='selftests',
-          python_requires='>=2.6')
+          python_requires='>=2.6',
+          install_requires=['stevedore'])
```

Comment:
The problem is, when someone installs from pip, if not all the deps are properly stated, we might end up with a lot of broken functionality...